### PR TITLE
Latest configs with cleanup

### PR DESCRIPTION
**Pull request by ethanglaser**

### CODE REVIEW

File: configs/spmd/large_scale/large_scale.json [hunk 1]
```diff
@@ -159,24 +64,13 @@
...
-                "dtype": "float64",
-                "distributed_split": "rank_based"
-            },
-            "bench": {
-                "mpi_params": {"n": [1,2,6,12,24,48,96,192,384], "ppn": 12, "-hostfile": "", "-cpu-bind=list:0-7,104-111:8-15,112-119:16-23,120-127:24-31,128-135:32-39,136-143:40-47,144-151:52-59,156-163:60-67,164-171:68-75,172-179:76-83,180-187:84-91,188-195:92-99,196-203": "--envall gpu_tile_compact.sh" }
-            }
-        },
-
-        "large scale strong two nodes parameters": {
+        "large scale strong <64 parameters": {
```
Comment (md-shafiul-alam) [inline configs/spmd/large_scale/large_scale.json:67]: <=

File: configs/spmd/large_scale/large_scale.json [hunk 2]
```diff
@@ -81,75 +36,25 @@
...
-
-        "large scale 2048 parameters": {
+        "large scale >64 parameters": {
             "data": {
                 "dtype": "float64",
                 "distributed_split": "None"
             },
             "bench": {
-                "mpi_params": {"n": [24576], "ppn": 12, "-hostfile": "", "-cpu-bind=list:0-7,104-111:8-15,112-119:16-23,120-127:24-31,128-135:32-39,136-143:40-47,144-151:52-59,156-163:60-67,164-171:68-75,172-179:76-83,180-187:84-91,188-195:92-99,196-203": "--envall gpu_tile_compact.sh" }
+                "mpi_params": {"n": [768,1536,3072,6144,12288,24576], "ppn": 12, "-hostfile": "", "-cpu-bind=list:0-7,104-111:8-15,112-119:16-23,120-127:24-31,128-135:32-39,136-143:40-47,144-151:52-59,156-163:60-67,164-171:68-75,172-179:76-83,180-187:84-91,188-195:92-99,196-203": "--envall gpu_tile_compact.sh" }
```
Comment (md-shafiul-alam) [inline configs/spmd/large_scale/large_scale.json:54]: >=, 768 is common in both configs < and >

Comment (ethanglaser) [reply configs/spmd/large_scale/large_scale.json]: removed >64 entirely since its just used for experimentation

**Review by md-shafiul-alam — APPROVED** [review]
spmd looks good, I will check to clean the online and bf16 parts. let's get it merged

**Pull request merged**